### PR TITLE
Show incomplete/skipped bugs, show tooltip on progress bars

### PR DESCRIPTION
**Pull request by Martchus**
Implement https://progress.opensuse.org/issues/13910

### CODE REVIEW

File: lib/OpenQA/WebAPI/Plugin/Helpers.pm [hunk 1]
```diff
@@ -254,6 +254,30 @@ sub register {
...
+                        $res . ' ' . $key;
+                    });
+            }
+            return '';
+        });
+
+    $app->helper(
+        build_progress_bar_title => sub {
+            my ($c, $res) = @_;
+            my @keys = ('passed', 'unfinished', 'softfailed', 'failed', 'incomplete', 'skipped', 'total',);
```
Comment (coolo) [inline lib/OpenQA/WebAPI/Plugin/Helpers.pm]: You really have a talent on avoiding the cool features and abusing the abstruse features of perl :)
```
my @keys = qw(passed unfinished ..

Comment (okurz) [inline lib/OpenQA/WebAPI/Plugin/Helpers.pm]: … with the help of @asdil12 and me ;-)

**Comment by coolo** [comment]
the addition of skipped I agree with, but I don't like the split between incomplete and failed. This is too much detail for this entrance point.

**Comment by Martchus** [comment]
@okurz If you don't see a reason why we shouldn't combine failures and incompletes, I'll change it in the way coolo proposed.

**Comment by Martchus** [comment]
Rebased and fixed, @okurz agrees.

**Comment by coolo** [comment]
tidy -> :+1:

**Pull request merged**